### PR TITLE
feat(JobMonitor): Expose Job log

### PR DESCRIPTION
**Pull request by cjh1**
Add panel to display any job log entries

### CODE REVIEW

**Comment by cjh1** [comment]
@TristanWright PTAL

File: src/panels/JobMonitor/ExecutionUnit.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            open={false}
+          >
+            <pre className={ style.log }>
+            { // reduce log array to a string with formatted entries
+              this.props.unit.log.reduce((prevVal, entry, index) => {
+                var content = prevVal;
+                content += `[${formatTime(entry.created)}] ${entry.levelname}: ${entry.msg}\n`;
+
+                if (entry.exc_info) {
+                  content += entry.exc_info[2].join('');
```
Comment (TristanWright) [inline src/panels/JobMonitor/ExecutionUnit.js]: Is `entry.exc_info[2]` ever an array?

File: src/panels/JobMonitor/ExecutionUnit.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      return (
+        <section key={this.props.unit._id} className={ style.listItem }>
+          <strong className={ style.itemContent }>{this.props.unit.name.split('.').pop()}</strong>
+          <div className={ style.itemContent }>{this.props.unit.status}</div>
+        </section>
+      );
+    }
+
+    return (
+        <section key={this.props.unit._id} className={ style.logListItem }>
```
Comment (TristanWright) [inline src/panels/JobMonitor/ExecutionUnit.js]: `keys={}` should be where the element repeats [here](https://github.com/Kitware/HPCCloud/pull/257/commits/42765b06048d08149985c7e754d33ec407252338#diff-c489ca458627ef15cb8c294d34d6cd1aR62)

File: src/panels/JobMonitor/ExecutionUnit.js (hunk 1 above)
Comment (cjh1) [inline src/panels/JobMonitor/ExecutionUnit.js]: yes, it the array of stack trace elements.

**Comment by cjh1** [comment]
@TristanWright  PTAL

**Comment by TristanWright** [comment]
looks great, merging.

**Pull request merged**